src/swagger-specs/destinations.yaml

```diff
@@ -973,7 +970,7 @@ paths:
         When performing listing (GET) requests in the Flow Service API, you can use query parameters to [sort and filter responses](https://experienceleague.adobe.com/docs/experience-platform/dataflows/api/sort-and-filter.html?lang=en).
```

Comment:
Not related to your work but i spotted another '?lang=en' maybe do a quick find and delete for the file.

Reply:
yep, there were actually 20+ instances of `?lang=en` that I removed now